pkg/sql/tests/kv_test.go

```diff
@@ -297,3 +312,47 @@ func BenchmarkKV(b *testing.B) {
 		})
 	}
 }
+
+// BenchmarkTracing measures the overhead of always-on tracing. It also reports
+// the memory utilization.
+func BenchmarkTracing(b *testing.B) {
```

Comment:
Will this run during `make bench`? If so, this is likely overkill. We don't care much about comparisons between the tracing modes once the default is flipped (since we have no intention of going back). If we want to keep the different modes around during the transition period that's fine by me, but you can just make a one-line addition of `newKVSQLWithTracing` here:

https://github.com/cockroachdb/cockroach/blob/f54afe7f77223929f40bae8b2e76e7deb7b30569/pkg/sql/tests/kv_test.go#L273

and then we don't have to duplicate the whole benchmark.